server/athenian/api/models/precomputed/models.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime, timezone
+
+from sqlalchemy import CHAR, Column, func, Integer, LargeBinary, String, Text, TIMESTAMP
+from sqlalchemy.dialects.postgresql import HSTORE
+
+from athenian.api.models import create_base
+
+
+Base = create_base()
+
+
+class GitHubPullRequestTimes(Base):
+    """A group of repositories identified by an integer."""
+
+    __tablename__ = "github_pull_request_times"
+
+    pr_node_id = Column(CHAR(32), primary_key=True)
+    release_match = Column(Text(), primary_key=True)
+    repository_full_name = Column(String(64 + 1 + 100), nullable=False)
+    pr_created_at = Column(TIMESTAMP(timezone=True), nullable=False)
+    pr_done_at = Column(TIMESTAMP(timezone=True))
+    developers = Column(HSTORE(), nullable=False)
+    format_version = Column(Integer(), nullable=False, default=1, server_default="1")
+    data = Column(LargeBinary())
```

Comment:
What will be the content of this?

Reply:
I plan to `pickle.dumps` our `PullRequestTimes` there. We can add more data in the future, including the precomputed metrics - but not with the current arch.